docs/src/languages/php/fpm.md

```diff
@@ -83,21 +120,29 @@ As an example, consider the following output:
       6 131072
 ```
 
-This indicates that the majority of requests (4800) used 2048 KB of memory.
-In this case that's likely application caching at work.
-Most requests used up to around 10 MB of memory, while a few used as much as 18 MB and a very few (6 requests) peaked at 131 MB.
-(In this example those are probably cache clears.)
+In the above output:
+
+- The majority of requests (4800) peaked at 2048 KB of memory, 
+  likely thanks to application caching.
+- Most other requests (3206) used up to around 10 MB of memory.
```

Comment:
:+1: Clearer as a list, for sure.

:clipboard: https://developers.google.com/style/numbers#commas-and-decimal-points-in-numbers
```suggestion
- The majority of requests (4,800) peaked at 2,048 KB of memory, 
  likely thanks to app caching.
- Most other requests (3,206) used up to around 10 MB of memory.
```